.gitlint

```diff
@@ -0,0 +1,2 @@
+[general]
+ignore=B6
```

Comment:
I've disabled the rule to ignore empty commit body. These are the available [rules](https://jorisroovers.com/gitlint/rules/)